scaling_performance/optimizing_storage.adoc

```diff
@@ -86,8 +86,7 @@ The following table summarizes the recommended and configurable storage technolo
 | Yes
 | Configurable
 | Configurable
-| Configurable footnoteref:[metrics-warning,For metrics, it is an anti-pattern to use any shared storage and a single volume
-(RWX). By default, metrics deploys with one volume per Cassandra Replica.]
+| Configurable
```

Comment:
Add back this: _footnoteref:[metrics-warning,For metrics, it is an anti-pattern to use any shared storage and a single volume (RWX)._

Reply:
Ultimately adopted this in 4.x: _footnoteref:[metrics-warning,For metrics, using file storage with the ReadWriteMany (RWX) access mode is unreliable. If you use file storage, do not configure the RWX access mode on any PersistentVolumeClaims that are configured for use with metrics.]_